x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/FilterLongVector.java

```diff
@@ -1,105 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-package org.elasticsearch.compute.data;
-
-import org.apache.lucene.util.RamUsageEstimator;
-import org.elasticsearch.core.Releasables;
-
-/**
- * Filter vector for LongVectors.
- * This class is generated. Do not edit it.
- */
-public final class FilterLongVector extends AbstractFilterVector implements LongVector {
-
-    private static final long BASE_RAM_BYTES_USED = RamUsageEstimator.shallowSizeOfInstance(FilterLongVector.class);
-
-    private final LongVector vector;
-
-    private final LongBlock block;
-
-    FilterLongVector(LongVector vector, int... positions) {
-        super(positions, vector.blockFactory());
-        this.vector = vector;
-        this.block = new LongVectorBlock(this);
-    }
```

Comment:
++ to removing the `X-FilterVector`s; the way their references behaved made ownership hard to reason about.

thought: Once we introduce refcounting, I think we'll want to bring back the essential logic from this, i.e. take an existing vector + a positions array/bitmap to create a filtered, shallow copy of an existing vector. But that can probably go directly into `X-Vector` or `X-ArrayVector` then.